node.hcl

```diff
@@ -6,7 +6,7 @@ strip = 1
 env = {
   "COREPACK_HOME": "${HERMIT_ENV}/.hermit/node",
   "NPM_CONFIG_PREFIX": "${HERMIT_ENV}/.hermit/node",
-  "NPM_CONFIG_CACHE": "${HERMIT_ENV}/.hermit/node/cache",
```

Comment:
Unfortunately there's a bug in recursive substitution where it doesn't always work. This will need to be the same as the other lines.

Reply:
Done!